libs/dh/shared/feature-authorization/src/lib/dh-actor-token.service.ts

```diff
@@ -138,6 +149,6 @@ export class DhActorTokenService {
   }
 
   private isTokenRequest(request: HttpRequest<unknown>): boolean {
-    return request.url.endsWith('/v1/Token');
+    return request.url.includes('/v1/Token');
```

Comment:
suggestion: instead of hardcoding the value, maybe we can use `this.apiToken.getTokenUrl`. The same applies to `isUserActorsRequest` method.

Reply:
https://github.com/Energinet-DataHub/greenforce-frontend/pull/3438/commits/193d510cd4965327910f62d7f54eca3f8c3dbdd8Good idea, fix in